templates/footer.twig

```diff
@@ -1 +1,23 @@
-<span class="copyright">Copyright {{"now"|date('Y')}} Greenpeace International</span>
+  <div id="footer">
+    <div class="container">
+      <div class="row justify-content-center">
+        <div class="col-md-6">
+          <ul class="footer-social-media list-unstyled">
+            {% for social in footer_social_menu %}
+              <li><a href="{{ social.url }}"><i class="{% for class_name in social.classes %}{{ class_name }} {% endfor %}"></i></a></li>
+            {% endfor %}
+          </ul>
+          <ul class="list-unstyled footer-links">
+            {% for primary in footer_primary_menu %}
+              <li><a href="{{ primary.url }}">{{ primary.title }}</a><span class="hidden-md-down">/</span></li>
+            {% endfor %}
+            {% for secondary in footer_secondary_menu %}
+              <li><a href="{{ secondary.url }}">{{ secondary.title }}</a><span class="hidden-md-down">/</span></li>
+            {% endfor %}
+          </ul>
+          <p class="copyright-text">&copy; {{ copyright_text|raw }}</p>
+          <p class="text-center">Greenpeace {{"now"|date('Y')}}</p>
+        </div>
+      </div>
+    </div>
+  </div>

```

Comment:
Needs escaping of vars used in HTML - URLs, attributes and text.

Reply:
Hello @dashaluna we will be using the Autoescape strategy, so all fields will be escaped by default unless we use the |raw filter on some which we will have first passed from sanitize or wp_kses functions.

Reply:
@angtheod cool, thanks for letting me know.